config/sources/states.js

```diff
@@ -255,6 +255,7 @@ module.exports = {
       description: 'Daily Difference in totalTestResults',
       nullable: true,
       example: '',
+      sourceFunction: (item) => 0,
```

Comment:
This will make it always return 0, how about:

`sourceFunction: (item) => item.totalTestResultsIncrease ? item.totalTestResultsIncrease : 0`

Reply:
When I tested it (turned out getting an environment working was easy!), it seemed like it was setting it to 0 as a default, and then `addIncrease()` was coming along later and replacing the zero for everything except the oldest row where null was getting left behind.

But spelling that out in the sourceFunction is cleaner. Will do.